scm/doc/TechGuide/chap_quick.tex

```diff
@@ -109,19 +109,22 @@ \section{Compiling SCM with CCPP}
 \label{section: compiling}
 The first step in compiling the CCPP and SCM is to match the physics variables (between what the host model -- SCM -- can provide and what is needed by physics schemes in the CCPP), and build the physics caps needed to use them.  Following this step, the top level build system will use \execout{cmake} to query system parameters and \execout{make} to compile the components.  A platform-specific script is provided to load modules and set the user environment for common platforms.  If you are not using one of these platforms, you will need to set up the same environment on your platform.
 \begin{enumerate}
-    \item Run the CCPP prebuild script to match required physics variables with those available from the dycore (SCM) and to generate physics caps and makefile segments.
+    \item Run the CCPP prebuild script to match required physics variables with those available from the dycore (SCM) and to generate physics caps and makefile segments. The default behavior is to use the so-called ``dynamic'' mode where software caps for all schemes in the CCPP repository are generated and a dynamic, shared library is generated for the CCPP physics. The basic syntax is
 \begin{lstlisting}[language=bash]
 ./ccpp/framework/scripts/ccpp_prebuild.py --config=./ccpp/config/ccpp_prebuild_config.py [--debug]
 \end{lstlisting}
+A second, more performant ``static'' mode is available that generates software caps for each physics group defined in the supplied SDFs and generates a static library that becomes part of the SCM executable. Using this mode, flexibility of defining suites at runtime is lost, although one can choose among a set of suites defined at compilation time. To use this mode, the syntax becomes
+\begin{lstlisting}[language=bash]
+./ccpp/framework/scripts/ccpp_prebuild.py --config=./ccpp/config/ccpp_prebuild_config.py --static --suites=suite list [--debug]
+\end{lstlisting}
+where the supplied suite list is a comma-separated list of suite names that exist in the ./ccpp/suites directory.
```

Comment:
I think we need to say here that the suite names are the suite definition files minus the suite_ prefix and minus the .xml suffix. Maybe add a line showing an example for three suites? This reminds me that we want to have a better solution for building the code, which involves running ccpp_prebuild.py as part of the cmake command. For the static build, all suites in the ccpp/suites directory should be compiled into the executable so that the multi-run script works.

Reply:
Great suggestion, thanks. I added three example suite names and mentioned what you suggested.